runtime/fundamentals/modules.md

```diff
@@ -0,0 +1,450 @@
+---
+title: "Using modules in Deno"
+oldUrl: 
+  - /runtime/manual/basics/modules/
+  - /runtime/manual/basics/modules/integrity_checking/
+  - /runtime/manual/basics/modules/module_metadata/
+  - /runtime/manual/basics/modules/proxies/
+  - /runtime/manual/basics/modules/publishing_modules/
+  - /runtime/manual/basics/modules/reloading_modules/
+  - /runtime/manual/advanced/publishing/dnt/
+  - /runtime/manual/advanced/publishing/
+  - /runtime/manual/basics/vendoring/
+  - /runtime/manual/node/cdns.md
+  - /runtime/manual/advanced/http_imports/
+---
+
+Deno standardizes the way modules are imported in both JavaScript and TypeScript
+using the
+[ECMAScript module standard](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
+by default. In line with this standard, **file names must be specified in
+full**. You should not omit the file extension and there is no special handling
+of `index.js`.
```

Comment:
This belongs in its own section - not part of the introduction. Also this is specific to local modules - it does not apply to https modules, npm modules, or jsr modules.

Something like:

> ## Local imports
> 
> To link to code in your project, use relative imports. These always start with "./" or "../" and end in a file extension. For example... 

After this section about local imports then add a section about remote imports aka third party modules - third party modules always start like this: `https://` or `npm:` or `jsr:`

Once these are explained, then talk about bare specifiers and import maps. Followed by more detailed information.